tests/integration/helpers.bash

```diff
@@ -14,7 +14,15 @@ BUSYBOX_IMAGE="$BATS_TMPDIR/busybox.tar"
 BUSYBOX_BUNDLE="$BATS_TMPDIR/busyboxtest"
 
 # hello-world in tar format
-HELLO_IMAGE="$TESTDATA/hello-world.tar"
+case $(dpkg --print-architecture) in
+arm64)
+	HELLO_IMAGE="$TESTDATA/hello-world-aarch64.tar"
+;;
+*)
+	HELLO_IMAGE="$TESTDATA/hello-world.tar"
+;;
```

Comment:
It isn't necessary to have two different filenames for this.

Reply:
@cyphar I uploaded the file of 'hello-world-aarch64.tar' in 'runc/tests/integration/testdata/'.

The file of 'hello-world.tar' can't be used on Arm64 platform, because the binary files with x86 format can't be used on Arm platform.

Reply:
Ah, I didn't notice that you included the `.tar` in `testdata` (I thought we always downloaded the rootfs archive).